### DISCUSSION TITLE
Search for artist?

### DISCUSSION THREAD

**Discussion by fromCologne** [hklages/node-red-contrib-sonos-plus#154] [Q&A] [answered]
Can I search for an artist?
With library.get.albums I can fill a dashboard list with albums to choose from.
But I don´t find a similar function to search for artists.

**Comment by hklages** [comment]
That piece is missing. Do you use mysonos (library.export.xxx) or universal node (group.play.library.xxx)? 

I could implement it by end of this week.

**Comment by fromCologne** [comment]
Dear Henning,

until yesterday I used the two nodes with mysonos and than export.
Now I installed 5.1.0 and changed to one node group.play.library...
Album search based on node library.get.album following a table node to choose from. I use a central raspi with touchscreen in my livingroom.
Idea is to use another table to choose the artist. Or maybe a combination.

> could implement it by end of this week
Don't panic ;-)
I must debug a strange behavior of the virtual keyboard node first.

See you
Ingo



Am 18. Mai 2021 07:55:51 MESZ schrieb "H. Klages" ***@***.***>:

**Comment by hklages** [comment]
... 5.1.1 hat jetzt group.play.library.artist. Habe es mit (Udo) Jürgens
getestet und funktioniert.
Melde Dich, wenn du mehr brauchst.

Grüße aus München nach Köln.
Henning

**Comment by fromCologne** [comment]
Hallo Henning,
 
5.1.1. installiert und funktioniert perfekt.
(Udo) J&uuml;rgens findet bei mir nichts, aber (Udo) Lindenberg bringts ;-)
Perfekte Arbeit. Super Danke.
 
> Melde Dich, wenn du mehr brauchst.
Ja, wenn Du so fragst...
group.play.library.artist findet den ersten Match. Eine coole Erweiterung w&auml;re noch library.get.artist beim MySonos Node.
Dann k&ouml;nnte man daraus so wie f&uuml;r Alben via JSONata auch eine klickbare Tabelle mit K&uuml;nstlern erstellen.
 
Mehr Koffein ist auf dem Weg :-)
 
Gr&uuml;&szlig;e aus K&ouml;ln
Ingo
 


-----urspr&uuml;ngliche Nachricht-----
 Von:  ***@***.*** ***@***.***" target="_blank" rel="noopener)
 Gesendet: 19.05.2021 13:17 Uhr
 An:  ***@***.*** ***@***.***" target="_blank" rel="noopener)
 Kopie:  ***@***.*** ***@***.***" target="_blank" rel="noopener),  ***@***.*** ***@***.***" target="_blank" rel="noopener)
 Betreff: Re: [hklages/node-red-contrib-sonos-plus] Search for artist? (#154)


 
... 5.1.1 hat jetzt group.play.library.artist. Habe es mit (Udo) J&uuml;rgens
getestet und funktioniert.
Melde Dich, wenn du mehr brauchst.

Gr&uuml;&szlig;e aus M&uuml;nchen nach K&ouml;ln.
Henning


On Tue, May 18, 2021 at 10:40 AM fromCologne ***@***.***>
wrote:

> Dear Henning,
>
> until yesterday I used the two nodes with mysonos and than export.
> Now I installed 5.1.0 and changed to one node group.play.library...
> Album search based on node library.get.album following a table node to
> choose from. I use a central raspi with touchscreen in my livingroom.
> Idea is to use another table to choose the artist. Or maybe a combination.
>
> > could implement it by end of this week
> Don't panic ;-)
> I must debug a strange behavior of the virtual keyboard node first.
>
> See you
> Ingo
>
>
>
> Am 18. Mai 2021 07:55:51 MESZ schrieb "H. Klages" ***@***.***>:
> >That piece is missing. Do you use mysonos (library.export.xxx) or
> >universal node (group.play.library.xxx)?
> >
> >I could implement it by end of this week.
> >
> >
> >--
> >You are receiving this because you authored the thread.
> >Reply to this email directly or view it on GitHub:
> >
> https://github.com/hklages/node-red-contrib-sonos-plus/discussions/154#discussioncomment-751153
>
> &mdash;
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> &lt;https://github.com/hklages/node-red-contrib-sonos-plus/discussions/154#discussioncomment-751722>,
> or unsubscribe
> &lt;https://github.com/notifications/unsubscribe-auth/AEDZCHZ7G4W2SMKIJ3QQVOLTOIRYHANCNFSM45AU2YDQ>
> .
>
&mdash;
You are receiving this because you authored the thread.
Reply to this email directly,  view it on GitHub (https://github.com/hklages/node-red-contrib-sonos-plus/discussions/154#discussioncomment-757782" target="_blank" rel="noopener), or  unsubscribe (https://github.com/notifications/unsubscribe-auth/ABXKLBZP2JMKSH23TWW2WVTTOOM5XANCNFSM45AU2YDQ" target="_blank" rel="noopener).<img src="https://github.com/notifications/beacon/ABXKLB7NZHPV7MQVZDDCUDTTOOM5XA5CNFSM45AU2YD2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAAFZAFQ.gif" alt="" width="1" height="1" />

 -----urspr&uuml;ngliche Nachricht Ende-----

**Comment by hklages** [comment]
Hallo Ingo,

Danke für den Kaffee! Hoffentlich kommt jetzt bald das gute Wetter um ihn
draußen zu genießen.

library.get.artist habe ich auf die ToDo Liste gesetzt und wird mit 5.1.2
kommen - wahrscheinlich Ende nächster Woche.

Bis dann
Henning

**Comment by hklages** [comment, marked as answer]
Hi - 5.2.4 hat jetzt auch bzgl. my sonos artists/albums/playlists/tracks.

**Reply by fromCologne** [reply]
Hallo Henning,
coole Sache das. Funzt super.
Grüsse
Ingo
